tests/playwright/tests/sdk/messages/multi.spec.js

```diff
@@ -46,3 +46,11 @@ messageTest.describe('Multi Messages', () => {
         await messageAxeCoreScan(messageIframe);
     });
 });
+messageTest.describe('Flex Test', () => {
+    messageTest('Flex', async ({ navigatePage, page, loadMessage, messageAxeCoreScan }) => {
+        await navigatePage({ account: 'DEV0USGENERIC', url: '/accessibility/flexsdk.html' });
+        const messageIframe = await loadMessage();
+        await page.waitForTimeout(5000);
```

Comment:
```suggestion
        await page.waitForTimeout(5000);
        const messageIframe = await loadMessage();
```
can we switch these two lines as the `flexsdk.html` doesn't load all the banners yet before it grabs the `loadMessage`